doc/nrf/releases_and_maturity/releases/release-notes-2.7.0.rst

```diff
@@ -0,0 +1,1238 @@
+.. _ncs_release_notes_270:
+
+|NCS| v2.7.0 Release Notes
+##########################
+
+.. contents::
+   :local:
+   :depth: 3
+
+|NCS| delivers reference software and supporting libraries for developing low-power wireless applications with Nordic Semiconductor products in the nRF52, nRF53, nRF54, nRF70, and nRF91 Series.
+The SDK includes open source projects (TF-M, MCUboot, OpenThread, Matter, and the Zephyr RTOS), which are continuously integrated and redistributed with the SDK.
+
+Release notes might refer to "experimental" support for features, which indicates that the feature is incomplete in functionality or verification, and can be expected to change in future releases.
+To learn more, see :ref:`software_maturity`.
+
+Highlights
+**********
+
+HIGHLIGHTS TBD
+
+Sign up for the `nRF Connect SDK v2.7.0 webinar`_ to learn more about the new features.
+
+See :ref:`ncs_release_notes_270_changelog` for the complete list of changes.
+
+Release tag
+***********
+
+The release tag for the |NCS| manifest repository (|ncs_repo|) is **v2.7.0**.
+Check the :file:`west.yml` file for the corresponding tags in the project repositories.
+
+To use this release, check out the tag in the manifest repository and run ``west update``.
+See :ref:`cloning_the_repositories` and :ref:`gs_updating_repos_examples` for more information.
+
+For information on the included repositories and revisions, see `Repositories and revisions for v2.7.0`_.
+
+IDE and tool support
+********************
+
+`nRF Connect extension for Visual Studio Code <nRF Connect for Visual Studio Code_>`_ is the recommended IDE for |NCS| v2.7.0.
+See the :ref:`installation` section for more information about supported operating systems and toolchain.
+
+Supported modem firmware
+************************
+
+See `Modem firmware compatibility matrix`_ for an overview of which modem firmware versions have been tested with this version of the |NCS|.
```

Comment:
Earlier we  had one comp matrix (for nRF9160) to provide modem firmware version compatibility. 
But, now we also have for nRF9161 as well : https://docs.nordicsemi.com/bundle/comp_matrix_nrf9161/page/COMP/nrf9161/nrf9161_modem_fw.html

I think this one also needs to be mentioned in the release notes. 
```suggestion
See the following documentation for an overview of which modem firmware versions have been tested with this version of the |NCS|.

* `Modem firmware compatibility matrix for the nRF9160 DK`_
* `Modem firmware compatibility matrix for the nRF9161 DK`_
```
The link.txt for `Modem firmware compatibility matrix`_   needs to be updated with `Modem firmware compatibility matrix for the nRF9160 DK`
Also, need to add a link for `Modem firmware compatibility matrix for the nRF9161 DK` (https://docs.nordicsemi.com/bundle/comp_matrix_nrf9161/page/COMP/nrf9161/nrf9161_modem_fw.html)

@lemrey  - Please confirm.

Reply:
That's correct, plus this release saw the introduction of yet another firmware that is called: `DECT NR+ PHY firmware`. It's not a "modem" firmware. At the modem it is unsure whether the compatibility matrix for NCS and that firmware will be in the same page (https://docs.nordicsemi.com/bundle/comp_matrix_nrf9161/page/COMP/nrf9161/nrf9161_modem_fw.html) or whether they will be separate.